operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3618,6 +3618,193 @@
       "timeShift": null,
       "title": "Watcher Latency",
       "type": "timeseries"
+    },
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 179
+      },
+      "id": 79,
+      "panels": [],
+      "repeat": "datasource",
+      "repeatDirection": "h",
+      "title": "Tekton Chains",
+      "type": "row"
+    },
+    {
+      "datasource": null,
```

Comment:
Although, the panels look good to me,  I noticed we're missing the `description` fields where we usually describe/ give context for the expression used in the query. This will also add to the consistency since we've a description in other dashboards. For e.g.- https://github.com/openshift-pipelines/pipeline-service/pull/709/files#diff-13716f85d071b7d6b050dd4f5d738d85f4627f17f20ee8b68fd8f39084c72ce9R3058

Reply:
We can safely skip this since the dashboard is not showing any useful metrics.